visualization/app/codeCharta/state/effects/updateVisibleTopLabels/updateVisibleTopLabels.effect.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { TestBed } from "@angular/core/testing"
+import { EffectsModule } from "../../angular-redux/effects/effects.module"
+import { ApplicationInitStatus } from "@angular/core"
+import { UpdateVisibleTopLabelsEffect } from "./updateVisibleTopLabels.effect"
+import { setAmountOfTopLabels } from "../../store/appSettings/amountOfTopLabels/amountOfTopLabels.actions"
+import { codeMapNodesSelector } from "../../selectors/accumulatedData/codeMapNodesSelector"
+import { Subject } from "rxjs"
+
+import { visibleFileStatesSelector } from "../../selectors/visibleFileStates.selector"
+import { FileState } from "../../../model/files/files"
+import { Store } from "../../angular-redux/store"
+import { CodeMapNode } from "../../../codeCharta.model"
+
+describe("updateVisibleTopLabelsEffect", () => {
+	let mockedVisibleFileStatesSelector = new Subject()
```

Comment:
[Optional] per convention subjects and observerables end with `$`

Reply:
You are right, but you also never use this convention in your effect tests. That's why I didn't do it either, for the purpose of consistency.:angel: